### ISSUE TITLE
Determine whether we could make this plugin support Scala.js

### ISSUE THREAD

**Issue by hussachai** [TailrecIO/sbt-aws-fn#4]
Not sure whether it is a good idea to include Scala.js into this project.
What do we have to consider?
- Minifying and merging (This is easy though)
- APIs. This is very hard to make it useful. AFAIK, we don't have Scala.js facade for AWS SDK.
  Actually, we don't have a lot of facades to make Scala.js application useable. 
  We can use `js.Dynamic.global`, but that'll be worse than using Javascript directly.

The project should have a flag to indicate the target platform -- JVM or NodeJs.
I think we should not mix both in the same project since it doesn't work in Lambda.

The plan will be pushed back until we get other JVM features implemented.

**Comment by ShaneDelmore** [comment]
I think this would be an amazingly useful feature as so much Scala code works with Scala.js these days and JS code has much better startup time on AWS Lambda.

**Comment by hussachai** [comment]
@ShaneDelmore Agreed. I'd love to add this feature for sure, but it's very hard at the moment to make Scala.js app useable on AWS Lambda. 
I've updated the description.
